### PR TITLE
New package: cmark-0.24.1

### PR DESCRIPTION
**Pull request by radare**

### CODE REVIEW

File: srcpkgs/cmark/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Template file for 'cmark'
+pkgname=cmark
+version=0.24.1
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
```
Comment (ebfe) [inline srcpkgs/cmark/template]: Looks like nothing is actually using this during the build.

Comment (radare) [inline srcpkgs/cmark/template]: Right! Its a devel dependency only

**Comment by radare** [comment]
Updated

**Comment by radare** [comment]
ready to merge

File: srcpkgs/cmark/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+homepage="https://github.com/jgm/cmark"
+distfiles="https://github.com/jgm/cmark/archive/${version}.tar.gz"
+checksum=7566d307ffff07567faa0381b248f60164eddcc0e0f5eb6e27d6f58e1269ab2a
+
+post_install() {
+	vlicense COPYING
+}
+
+cmark-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision} pkg-config"
```
Comment (ebfe) [inline srcpkgs/cmark/template]: drop pkg-config

**Comment by radare** [comment]
done

File: srcpkgs/cmark/template [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+cmark-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/include
+		vmove "usr/lib/*.so"
+		vmove "usr/lib/*.a"
+		vmove usr/lib/pkgconfig
```
Comment (xtraeme) [inline srcpkgs/cmark/template]: `vmove usr/share/man/man3`

File: srcpkgs/cmark/template [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+short_desc="CommonMark parsing and rendering library and program in C"
+maintainer="pancake <pancake@nopcode.org>"
+license="MIT"
+homepage="https://github.com/jgm/cmark"
+distfiles="https://github.com/jgm/cmark/archive/${version}.tar.gz"
+checksum=7566d307ffff07567faa0381b248f60164eddcc0e0f5eb6e27d6f58e1269ab2a
+
+post_install() {
+	vlicense COPYING
+	vmove usr/share/man/man3
```
Comment (Vaelatern) [inline srcpkgs/cmark/template]: Not this `vmove`

Comment (radare) [inline srcpkgs/cmark/template]: can you send a PR?

Comment (Vaelatern) [inline srcpkgs/cmark/template]: Just that vmove should be in the `pkg_install` of the `-devel` subpackage.

Comment (radare) [inline srcpkgs/cmark/template]: can you send a PR?

Comment (Vaelatern) [inline srcpkgs/cmark/template]: Here is a patch. PRs take a lot of work on my part. You can apply this with a simple `git am` (I think).

```
--- template	2016-03-27 01:03:28.054296905 -0400
+++ template	2016-03-27 01:03:43.597491228 -0400
@@ -12,7 +12,6 @@
 
 post_install() {
 	vlicense COPYING
-	vmove usr/share/man/man3
 }
 
 cmark-devel_package() {
@@ -23,5 +22,6 @@
 		vmove "usr/lib/*.so"
 		vmove "usr/lib/*.a"
 		vmove usr/lib/pkgconfig
+		vmove usr/share/man/man3
 	}
 }
```

Comment (radare) [inline srcpkgs/cmark/template]: Making a pr is just pressing the pencil button in the github web interface... But thanks, ill update as soon as possible

**Comment by radare** [comment]
Done, and squashed, anything else to merge?

File: srcpkgs/cmark/template (hunk 4 above)
Comment (Vaelatern) [inline srcpkgs/cmark/template]: Not for a person without write access. Thanks for updating!

Comment (radare) [inline srcpkgs/cmark/template]: you can create a pullrequest on ANY repository for ANY branch, you dont need write access to do that

Comment (Vaelatern) [inline srcpkgs/cmark/template]: Huh. Had not seen that feature. I'm used to git from the command line. To be fair, the patch was easiest for both of us.

**Pull request merged**